### ISSUE TITLE
Incompatible inrimage interface for car command

### ISSUE THREAD

**Issue by bereziat** [heimdali/heimdali#14]
car has the following interface:
`car  [-o <FILENAME>] [--] [--version] [-h] <IMAGE-IN>`
It should be:
`car <IMAGE-IN> <IMAGE-OUT> ....`
with capability to read stdin, write stdout

Actually, this issue adresses most of commands.

**Comment by dfroger** [comment]
Functional tests that `car` (and other command) must pass are:

```
car in.inr out.inr
car in.h5 out.h5
cat in.h5 | car > out.h5
cat in.h5 | car - > out.h5
```

Did I miss something?

**Comment by bereziat** [comment]
Le 03/03/15 10:20, Froger David a écrit :

> Functional tests that |car| (and other command) must pass are:
> 
> |car in.inr out.inr
> car in.h5 out.h5
> cat in.h5 | car > out.h5
> cat in.h5 | cat - > out.h5
> |

cat in.h5 | car - > out.h5
plutôt

> Did I miss something?
> non, c'est exhautif je crois
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dfroger/heimdali/issues/14#issuecomment-76911729.

## 

Dominique Béréziat
5 square des Champs Frétauts
91120 Palaiseau

**Comment by dfroger** [comment]
Fixed in 9d8c3bf
